src/main/java/com/github/manolo8/darkbot/backpage/HangarManager.java

```diff
@@ -72,6 +72,18 @@ public HangarResponse getCurrentHangar() {
         return currentHangar;
     }
 
+    /**
+     * @return The ID of the current hangar. A value of 0 indicates that the
+     *         currentHangar has not been initialized.
+     */
+    public int getCurrentHangarId() {
+        return getHangarList().getData().getRet().getHangars().stream()
```

Comment:
this can NPE in several places, hangar list can be null, data can be null, ret can be null, etc
While it's painful to write, you need to check for nullability every step of the way

Reply:
Solved as I would like to have at the moment "?."